queries/javascript/highlights.scm

```diff
@@ -113,23 +113,40 @@
 "." @punctuation.delimiter
 "," @punctuation.delimiter
 
-"--" @operator
-"-" @operator
-"-=" @operator
-"&&" @operator
-"+" @operator
-"++" @operator
-"+=" @operator
-"<" @operator
-"<<" @operator
-"=" @operator
-"==" @operator
-"===" @operator
-"=>" @operator
-">" @operator
-">>" @operator
-"||" @operator
-"??" @operator
+[
+  "--"
+  "-"
+  "-="
+  "&&"
+  "+"
+  "++"
+  "+="
+  "&="
+  "/="
+  "**="
+  "<<="
+  "||="
+  "&&="
+  "??="
+  "<"
+  "<="
+  "<<"
+  "="
+  "=="
+  "==="
+  "!="
+  "!=="
+  "=>"
+  ">"
+  ">="
+  ">>"
+  "||"
+  "??"
+  "%"
+  "%="
+] @operator
+
+(ternary_expression ["?" ":"] @operator)
```

Comment:
`?` is also an operator now. You can do `object?.value` (if object is null/undefined it will not throw and return undefined)

Reply:
Yeah, but the javascript parser doesn't support it so it throws an error :(. It would technically be a delimiter though right?

Reply:
`obj?.someProp` and null coalesing operator `??`

Reply:
it throws really ...? lets update the javascript parser then :) maybe issue this on the parser repo.

Reply:
I opened an issue on the typescript parser a while back but then realized it should go in the JS parser.

It says the nodes are unrecognized so the query fails to parse.

Reply:
I need to remove `??` for this to parse correctly :cry:

Reply:
jeez :(